claim_batch/services.py

```diff
@@ -267,31 +266,32 @@ def do_process_batch(audit_user_id, location_id, end_date):
             relative_price_g = False
             # assign start date accordin to product
             if product.period_rel_prices is not None:
-                start_date =  get_start_date(end_date, product.period_rel_prices)
+                start_date = get_start_date(end_date, product.period_rel_prices)
                 relative_price_g = True
             else:
                 if product.period_rel_prices_op is not None:
-                    start_date =  get_start_date(end_date, product.period_rel_prices_op)
+                    start_date = get_start_date(end_date, product.period_rel_prices_op)
                     # if only op is defiene fallback ip on the same periodicity
                     if product.period_rel_prices_ip is  None:
                         start_date_ip = start_date
                 if product.period_rel_prices_ip is not None:
-                    start_date_ip =  get_start_date(end_date, product.period_rel_prices_ip)
+                    start_date_ip = get_start_date(end_date, product.period_rel_prices_ip)
                     # if only ip is defiene fallback op on the same periodicity
-                    if product.period_rel_prices_op is  None:
+                    if product.period_rel_prices_op is None:
                         start_date = start_date_ip
             if start_date is None and start_date_ip is None:
                 # fall back on month
                 start_date = get_start_date(end_date, 'M')
                 relative_price_g = True
```

Comment:
to be move below , see other comment

Reply:
with the new approach, we can get rid of the ip/op stuff because it will be just 2 different payment plans

Reply:
that means we need a parameters in-patient/out-patient/both in the fee-for-service calcrule